stories/organisms/Dialogs.stories.tsx

```diff
@@ -93,3 +94,47 @@ export const _Confirm: ConfirmStory = {
         );
     },
 };
+
+export const _Dialog: ConfirmStory = {
+    name: 'Dialog',
+    args: {
+        title: 'Duplicate this page',
+        contentLabel: 'content label',
+        children: 'this is not as cool as I wanted ',
+    },
+    render: (args) => {
+        const [showAlert, setShowAlert] = React.useState(false);
+
+        const handleButtonClick = () => {
+            setShowAlert(true);
+            console.log('hel')
```

Comment:
prettier with onsave or something could avoid this Githib Actions / lint failures